packages/gunshi/src/decorators.ts

```
@@ -17,64 +17,72 @@ const EMPTY_RENDERER = async () => ''
  * Internal class for managing renderer decorators.
  * This class is not exposed to plugin authors.
  */
-export class Decorators<A extends Args = Args> {
-  #headerDecorators: RendererDecorator<string, A>[] = []
-  #usageDecorators: RendererDecorator<string, A>[] = []
-  #validationDecorators: ValidationErrorsDecorator<A>[] = []
-  #commandDecorators: CommandDecorator<A>[] = []
+export class Decorators<A extends Args = Args, E extends Record<string, unknown> = {}> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)